pages/x/index.tsx

```diff
@@ -17,18 +17,22 @@ const ThirdPartyRegistryList = () => {
     setQuery(event.target.value);
   }
 
-  const list = useMemo(
+  // Sorting the list only if entries are changed
```

Comment:
Sorting shouldn't actually be needed because `database.json` is already in alphabetical order.

Reply:
@lucacasonato  Is there any automated job to update the list in database.json?

I don't see any. Devs might end up in writing anywhere in the list. 

<img width="2032" alt="Screenshot 2020-06-20 at 6 10 38 AM" src="https://user-images.githubusercontent.com/2510560/85187398-1048f000-b2bd-11ea-9ece-53e4cad4dae4.png">

Reply:
There should be a test for it.

Reply:
Nope, there is none. There is a prettier linting check, that's it.

Reply:
https://github.com/denoland/deno_website2/blob/master/util/database.test.ts#L50-L53

Reply:
> https://github.com/denoland/deno_website2/blob/master/util/database.test.ts#L50-L53

Removing the sorting operation. Updated the PR.